charts/scalardb-mcp-server/values.yaml

```diff
@@ -0,0 +1,256 @@
+    # JWT Key Management Configuration
+    # ========================================
+    # RSA private key in PEM format (PKCS#8) - MUST be provided via secret
+    scalar.mcp.db.oauth.security.jwt.private-key=${SCALAR_MCP_DB_OAUTH_SECURITY_JWT_PRIVATE_KEY}
+
+    # RSA public key in PEM format (X.509) - MUST be provided via secret
+    scalar.mcp.db.oauth.security.jwt.public-key=${SCALAR_MCP_DB_OAUTH_SECURITY_JWT_PUBLIC_KEY}
+
+    # ========================================
+    # ScalarDB Connection Configuration
+    # ========================================
+    scalar.mcp.db.server.tool.mode=${SCALAR_MCP_DB_SERVER_TOOL_MODE:SQL}
+
+    # ScalarDB Cluster connection settings (credentials provided by user at login)
+    scalar.db.transaction_manager=${SCALAR_DB_TRANSACTION_MANAGER:cluster}
+    scalar.db.contact_points=${SCALAR_DB_CONTACT_POINTS:indirect:scalardb-cluster-envoy}
+    scalar.db.contact_port=${SCALAR_DB_CONTACT_PORT:60053}
+
+    # SQL connection configuration (inferred from cluster config)
+    scalar.db.sql.connection_mode=${scalar.db.transaction_manager}
+    scalar.db.sql.cluster_mode.contact_points=${scalar.db.contact_points}
+    scalar.db.sql.cluster_mode.contact_port=${scalar.db.contact_port}
+
+    # ========================================
+    # Logging Configuration
+    # ========================================
+    scalar.mcp.db.server.logging.level=${SCALAR_MCP_DB_SERVER_LOGGING_LEVEL:DEBUG}
+    scalar.mcp.db.server.logging.file.name=${SCALAR_MCP_DB_SERVER_LOGGING_FILE_NAME:}
+    logging.level.com.scalar.scalardb.mcp=${scalar.mcp.db.server.logging.level}
+    logging.file.name=${scalar.mcp.db.server.logging.file.name}
+    logging.threshold.console=on
+
+  # -- Environment variables for the container. Use this to inject secrets and configuration.
+  # @default -- No environment variables are set by default.
+  env: []
+    # Required secrets - MUST be provided for production:
+    # - name: SCALAR_MCP_DB_OAUTH_SESSION_CACHE_ENCRYPTION_KEY
+    #   valueFrom:
+    #     secretKeyRef:
+    #       name: scalardb-mcp-server-secrets
+    #       key: session-encryption-key
+    # - name: SCALAR_MCP_DB_OAUTH_SECURITY_JWT_PRIVATE_KEY
+    #   valueFrom:
+    #     secretKeyRef:
+    #       name: scalardb-mcp-server-secrets
+    #       key: jwt-private-key
+    # - name: SCALAR_MCP_DB_OAUTH_SECURITY_JWT_PUBLIC_KEY
+    #   valueFrom:
+    #     secretKeyRef:
+    #       name: scalardb-mcp-server-secrets
+    #       key: jwt-public-key
+    #
+    # Optional configuration overrides:
+    # - name: SCALAR_MCP_DB_OAUTH_ISSUER_URL
+    #   value: "https://mcp.example.com"
+    # - name: SCALAR_DB_CONTACT_POINTS
+    #   value: "indirect:scalardb-cluster-envoy.scalardb.svc.cluster.local"
+    # - name: SCALAR_MCP_DB_SERVER_LOGGING_LEVEL
+    #   value: "DEBUG"
+
+  service:
+    # -- Service type.
+    type: ClusterIP
+    # -- Service port.
+    port: 8080
+    # -- Container target port.
+    targetPort: 8080
```

Comment:
As I mentioned in another comment, I think it might be better to discuss and change the default port of ScalarDB MCP Server.